.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+**Note: new features are to be opened against next, hotfixes against master.**
+
+### Summary
+
+SUMMARY_GOES_HERE
+
+### Full changelog
```

Comment:
shouldn't we add a `changelog.md` in the root and have a checkbox here for updating that file? It would severely reduce the compilation effort of a release doc.